characters.py

```diff
@@ -56,14 +56,14 @@ def get_status(self):  # Since we are using some of the character variables, we
                "*" * 10)
         return msg
 
-    def xp_up(self, amount):
+    def xp_up(self, amount): # XP/Level up function, adds increases stats upon level up
         self.xp += amount
         if self.xp >= 100:
             self.xp -= 100
-            self.max_hp += random.randint(5,10)
-            self.attack += random.randint(2,5)
+            self.max_hp += random.randint(2,4)
```

Comment:
The logic increases each stat by a random amount each level, I don't see how a config file here would be of any benefit.